sdk/search/azure-search/src/main/java/com/azure/search/models/GeoPoint.java

```diff
@@ -116,27 +116,19 @@ public String toString() {
         return "";
     }
 
-    /**
```

Comment:
Why do we remove the getCoordinates?

Reply:
It was an ask to remove them, https://github.com/azure/azure-sdk-for-java/issues/8149